tests/serverpod_new_auth_test/serverpod_new_auth_test_server/test/integration/email_account_endpoint_test.dart

```diff
@@ -0,0 +1,411 @@
+import 'package:serverpod/serverpod.dart';
```

Comment:
@coderabbitai check this file, ensuring that all exposed methods of `EmailAccountEndpoint` are tested.

Double check that test descriptions match the implementation and check spelling.